build.gradle

```diff
@@ -56,6 +56,8 @@ project.ext.externalDependency = [
     'gmaTestModelsDataTemplate': "com.linkedin.datahub-gma:test-models-data-template:$gmaVersion",
     'gmaValidators': "com.linkedin.datahub-gma:validators:$gmaVersion",
     'graphqlJava': 'com.graphql-java:graphql-java:16.1',
+    'graphqlJavaSpringWebMvc': 'com.graphql-java:graphql-java-spring-boot-starter-webmvc:2.0',
+    'graphiqlSpring': 'com.graphql-java:graphiql-spring-boot-starter:5.0.2',
```

Comment:
This is used for graph visualization, correct? Is this necessary to include in DH?